server/index.js

```diff
@@ -1,5 +1,6 @@
 var express = require('express');
 var bodyParser = require('body-parser');
```

Comment:
Express v.4.x has [body-parser functionality](https://expressjs.com/en/api.html#express.json) in the form of a line like `app.use(express.json())`, so you probably don't need to install body-parser

Reply:
Good to know! Thanks!